### PR TITLE
fixing crash in DropletActivity

### PR DESCRIPTION
**Pull request by mohityv**
**Fixes** : #93 

**Changes** : Added if statements for checking if the response was successful and added NotNull annotation to response and call parameters of onResponse function.

### CODE REVIEW

**Comment by mohityv** [comment]
@championswimmer travis is failing because of some failed tests. Please look at it

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java [hunk 1]
```diff
@@ -93,16 +94,19 @@ public void onClick(View view) {
...
-                ImageView profilePic = ((ImageView) drawer.findViewById(R.id.accountPic));
-                if (profilePic != null && email != null && !email.isEmpty()) {
-                    Picasso.with(DropletActivity.this).load("https://www.gravatar.com/avatar/" + md5(email)).into(profilePic);
-                }
+            public void onResponse(@NonNull Call<AccountInfo> call, @NonNull Response<AccountInfo> response) {
+                String email = null;
+                if (response.isSuccessful() && response.body() != null) {
+                    email = response.body().getAccount().getEmail();
 
+                    if (((TextView) drawer.findViewById(R.id.accountEmail)) != null) {
```
Comment (4lch4) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java]: You should assign this to a TextView variable then check it for null so you don't have to run `findViewById()` twice as it's pretty costly in terms of performance.

```java
TextView accountEmailTV = (TextView) drawer.findViewById(R.id.accountEmail);

if (accountEmailTV != null) accountEmailTV.setText(email);
```

File: app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java [hunk 2]
```diff
@@ -93,16 +94,19 @@ public void onClick(View view) {
...
+                        ((TextView) drawer.findViewById(R.id.accountEmail)).setText(email);
+                    }
+                    ImageView profilePic = ((ImageView) drawer.findViewById(R.id.accountPic));
```
Comment (4lch4) [inline app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java]: How come you wrap the assignment in parenthesis? Does this achieve anything extra? 🤔

Couldn't you simply do `ImageView profilePic = (ImageView) drawer.findViewById(R.id.accountPic);`?

Comment (mohityv) [reply app/src/main/java/in/tosc/digitaloceanapp/activities/DropletActivity.java]: okay Ill remove the parenthesis

**Comment by mohityv** [comment]
@championswimmer @Alcha can you please review the pull now

**Comment by 4lch4** [comment]
Sorry for being so late to this issue, looks like Arnav took care of it.

**Pull request merged**